src/App.tsx

```diff
@@ -1,14 +1,42 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useState, useEffect, useMemo } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
+  const [todoList, setTodoList] = useState<Todo[]>([]);
+  const [searchWord, setSearchWord] = useState('');
+  const [filterOption, setFilterOption] = useState('all');
```

Comment:
move filter variants into an enum (all, active...) and use it instead of simple strings

Reply:
please use it created type
```suggestion
  const [filterOption, setFilterOption] = useState(FilteredBy.ALL);
```